docs/pages/upgrading/cloud-kubernetes.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Upgrade Teleport Cloud Clusters on Kubernetes
+description: Provides instructions for upgrading Teleport Cloud clusters that run on Kubernetes. 
+---
+
+This guide explains how to upgrade Teleport Cloud clusters running on
+Kubernetes. On Teleport Cloud, Auth Service and Proxy Service upgrades are
+managed for you. To keep agents up to date, you can either enroll them in
+automatic upgrades or upgrade them manually using the method you used to install
+Teleport.
+
+## Prerequisites
+
+- Familiarity with the [Upgrading Compatibility Overview](./overview.mdx) guide,
+  which describes the sequence in which to upgrade components of your cluster.
+
+- Teleport Enterprise Cloud or Teleport Team account. You can determine the
+  current version of these services by running the following command, where
+  `mytenant` is the name of your Teleport Team or Teleport Enterprise Cloud
+  tenant. This requires the [`jq` CLI tool](https://jqlang.github.io/jq/):
+
+  ```code
+  $ curl -s https://mytenant.teleport.sh/webapi/ping | jq '.server_version'
+  ```
+
+Teleport supports automatic agent updates for the `teleport-kube-agent` Helm
+chart. The [Automatic Update Architecture
+](../architecture/agent-update-management.mdx) guide describes how agent
+updating works. Automatic agent upgrades require:
+
+- A Teleport Cloud account that supports automatic upgrades. To determine if
+  your account supports automatic upgrades, run the following command, replacing
+  `mytenant.teleport.sh` with the address of your Teleport Cloud account:
+
+  ```code
+  $ curl -s https://mytenant.teleport.sh/webapi/ping | jq '.automatic_upgrades'
+  true
+  ```
+- At least one Teleport Enterprise agent deployed via the `teleport-kube-agent`
+  Helm chart.
+
+## Step 1/2. Determine whether any agents require automatic upgrades
+
+To determine if any agents in your cluster are not configured for automatic
+upgrades, run the following command. This prints a list of all Teleport services
+with no upgrader, filtering these to ignore the Auth Service and Proxy Service
+(which are maintained for you in Teleport Cloud):
+
+```code
+$ tctl inventory ls --upgrader=none | awk '!($3 ~ /(Proxy|Auth)/){print $0}'
+Server ID                            Hostname                        Services Version Upgrader
+------------------------------------ ------------------------------- -------- ------- --------
+00000000-0000-0000-0000-000000000000 teleport-agent-0                Kube     v13.4.3 none
+```
```

Comment:
@fspmarshall could we just lie and report a `managed` or `cloud` upgrader for cloud auths and proxies? or support negation in the `--services` filter so we can just exclude auth and proxy agents from the listing without resorting to awk?